ddtrace/contrib/botocore/patch.py

```diff
@@ -84,12 +86,15 @@ def patch():
     if getattr(botocore.client, "_datadog_patch", False):
         return
     botocore.client._datadog_patch = True
+    service = ext_service(pin=None, int_config=config.botocore)
```

Comment:
```suggestion
```
I think we don't actually need these lines, since everywhere just references the config anyways.